src/en/config-azure.md

```diff
@@ -1,123 +1,194 @@
-# Configuring for Windows Azure
+Title: Juju Azure provider  
 
-This process requires you to have a Windows Azure account. If you have not
-signed up for one yet, it can obtained at http://azure.microsoft.com/en-us/.
 
-You should start by generating a generic configuration file for Juju, using the
-command:
+# Configuring for Azure 
+
+The default behaviour is for units of a service to be allocated to
+machines in a service-specific Availability Set. Read the
+[Azure SLA](https://azure.microsoft.com/en-gb/support/legal/sla/) to learn how
+availability sets affect uptime guarantees.
+
+!!! Warning: Azure has two concurrent UI versions: the older "classic" console
+(https://manage.windowsazure.com) and the "new" console
+(https://portal.azure.com). The features necessary in this document are only
+available in the classic portal.
+
+
+## Prerequisites and installation of Juju
+
+ - An Azure account is required. See http://azure.microsoft.com.
+
+ - An SSL/TLS certificate, either an existing one or a new one, will be needed to
+   communicate with Azure.
+
+ - The Juju stable PPA is needed.
```

Comment:
This may be true right now, but it will become wrong. It is okay to say "Juju 2.0+ is required for the features discussed here" or something similar